client/src/app/components/Footer.tsx

```diff
@@ -36,7 +36,9 @@ function Tagline() {
 
 function Logo() {
   return (
-    <Anchor href="https://leastauthority.com" target="_blank">
+    <Anchor href="https://leastauthority.com" 
+      target="_blank" 
```

Comment:
Can we please not do this? Forcing new tabs/windows is against Web recommendations too, AFAIK.

Reply:
I am not against to change this, but can you be more specific which web recommendations? As far as I understand, this is more ethical question: give decision to users open in the same page or in new tab or decide for user, when to open in new tab/window.

Reply:
yes, if you say target=_blank your are removing the choice and forcing a new tab/window.

https://www.w3.org/TR/WCAG20-TECHS/G200.html

Reply:
I was trying to run some public wcag checker too, just currently didn't find one, which provides details results without registration. I will drop _blank